docs/transcript.rst

```diff
@@ -0,0 +1,161 @@
+========================
```

Comment:
This section is a very nice addition to the existing user manual.  It does a great job explaining what transcripts are, how to use them, and what some of the common pitfalls are and how to avoid them.  Well done!